docs/program-setup/serial/serial_communication.md

```diff
@@ -6,57 +6,60 @@ The Arm Mbed microcontroller on your board can communicate with a host PC over t
 
-There are many terminal applications for Windows, including:
+Mbed CLI has built-in serial support. To open a serial terminal when _one_ Mbed board is connected and using the default baud rate (9600):
 
-- [CoolTerm](http://freeware.the-meiers.org/) - this is the application we use in this example. We use it often because it usually "just works".
-- [Tera Term](http://sourceforge.jp/projects/ttssh2/files).
-- [PuTTY](http://www.chiark.greenend.org.uk/~sgtatham/putty/).
-- Some Windows PCs come with **Hyperterminal** installed.
+```
+mbed sterm
+```
 
-### Configuring the connection
+Messages printed after this point will be displayed, so to run the example application again, use the board's reset button or press Ctrl + B on the serial terminal. Now the application restarts and prints "Hello World!" to the terminal.
 
-1. Plug in your Mbed board.
-1. Open CoolTerm.
-1. Click **Connect**. This opens up an 8-n-1 9600 baud connection to the first available serial port. If you have more than one board plugged in, you may need to change the port under **Options > Serial Port > Port**.
+To exit the serial terminal, press Ctrl + C.
 
-Check your connection parameters:
+<span class="tips">**Tip:** To compile, flash and open a serial terminal in one command line, you can append `--flash --sterm` to `mbed compile`.</span>
 
-1. Select **Options > Serial Port**.
-1. You should see 9600 baud, 8 bits, 1 stop bit, no parity (9600-8-N-1).
-1. If you do not see your board, click **Re-Scan Peripherals**.
+## Additional options of `mbed sterm`
 
-Your terminal program is now configured and connected.
+More options are supported, run `mbed sterm -h` to show:
 
-## Using terminal applications on Linux
+    usage: mbed sterm [-h] [-m TARGET] [-p PORT] [-b BAUDRATE] [-e ECHO] [-r] [-v] [-vv]
 
-CoolTerm should work under Linux. If for some reason it doesn't, you can try one of the following:
+    Open serial terminal to connected target (usually board), or connect to a user-specified COM port
 
-- [Minicom](https://help.ubuntu.com/community/Minicom).
-- [GNU Screen](https://www.gnu.org/software/screen/manual/screen.html).
+    optional arguments:
+    -h, --help            show this help message and exit
+    -m TARGET, --target TARGET
+                            Compile target MCU. Example: K64F, NUCLEO_F401RE,
+                            NRF51822...
+    -p PORT, --port PORT  Communication port. Default: auto-detect. Specifying
+                            this will also ignore the -m/--target option above.
+    -b BAUDRATE, --baudrate BAUDRATE
+                            Communication baudrate. Default: 9600
+    -e ECHO, --echo ECHO  Switch local echo on/off. Default: on
+    -r, --reset           Reset the targets (via SendBreak) before opening
+                            terminal.
+    -v, --verbose         Verbose diagnostic output
+    -vv, --very_verbose   Very verbose diagnostic output
```

Comment:
Can you put this section between ``` for formatting?

Reply:
My IDE had issues rendering this particular quote when I used ```, so I thought markdown didn't allow some of the hyphen? I'll try again.

Reply:
Works now

Reply:
Done